protocol/v2/qbft/storage/ibft_store.go

```diff
@@ -32,17 +36,68 @@ type ParticipantsRangeEntry struct {
 	Identifier convert.MessageID
 }
 
+type Quorum struct {
+	Signers   []spectypes.OperatorID
+	Committee []spectypes.OperatorID
+}
+
+func (q *Quorum) ToSignersBitMask() SignersBitMask {
+	if len(q.Committee) > maxCommitteeSize || len(q.Signers) > maxCommitteeSize || len(q.Signers) > len(q.Committee) {
+		panic(fmt.Sprintf("invalid signers/quorum size: %d/%d", len(q.Committee), len(q.Signers)))
```

Comment:
we should probably return error instead of panicking, especially in an API triggered code.